cmd/create/service/cmd.go

```diff
@@ -419,7 +429,7 @@ func run(cmd *cobra.Command, argv []string) {
 		operatorIAMRoleList = append(operatorIAMRoleList, ocm.OperatorIAMRole{
 			Name:      operator.Name(),
 			Namespace: operator.Namespace(),
-			RoleARN:   getOperatorRoleArn(operatorRolesPrefix, operator, r.Creator),
+			RoleARN:   getOperatorRoleArn(operatorRolesPrefix, operator, r.Creator, path),
```

Comment:
Feel like this would solve the problem of creating the operator roles

Reply:
Looking at the other managed service operations, you're absolutely right! I don't think the current clients of managed services will use this feature, but since it doesn't affect the old way and helps maintain feature parity I think we can add support for it. Do you want to change the title of the pr to add support instead of blocking?

Reply:
Sure, let's test that then